chapters/equations.tex

```diff
@@ -518,6 +519,10 @@ \subsection{assert}\label{assert}
        AssertionLevel.warning);
 assert(T > 200 and T < 500, "Medium model outside feasible region");
 \end{lstlisting}
+
+It is recommended that models have a simple message as above and not include the value of any variables (like \lstinline!T!) in the message given to \lstinline!assert!.
+Tools are (especially for simple messages) recommended to automatically include the values of relevant variables, as part of the log or interactively accessible.
+Similarly tools are recommended to include the failed condition.
```

Comment:
This is already part of the normative text above, so it needs to be removed or reformulated with the user in focus:
```suggestion
In this case, for example, embedding the value of \lstinline!T! in the message is likely to result in duplicated information and should be avoided.
```

Reply:
The idea with "especially for simple messages" was to allow tools to skip the duplicated T in such cases.

Reply:
Basically to allow good compatibility with existing models that already have such complicated messages, but provide a simple recommendation for users.

Reply:
Ok, tools may be smarter for a transition

Reply:
This has now been reformulated to be clearer (for tools and users) and include the point about duplicate information.

Reply:
Resolving outdated conversation.